extensions/DynamicPageList/DPLMain.php

```diff
@@ -2168,7 +2168,7 @@ public static function dynamicPageList( $input, $params, $parser, &$bReset, $cal
 
 			if ( $bAddContribution ) {
 				$sSqlRCTable = $sRCTable . ' AS rc, ';
-				$sSqlSelPage .= ', SUM( ABS( rc.rc_new_len - rc.rc_old_len ) ) AS contribution, rc.rc_user_text AS contributor, rc.rc_user AS contributor_id'; // SUS-812
+				$sSqlSelPage .= ', SUM( ABS( rc.rc_new_len - rc.rc_old_len ) ) AS contribution, rc.rc_ip_bin AS contributor, rc.rc_user AS contributor_id'; // SUS-812
 				$sSqlWhere   .= ' AND page.page_id=rc.rc_cur_id';
 				if ($sSqlGroupBy != '') $sSqlGroupBy .= ', ';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)

[project-id]: wikia:app
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)

[project-id]: wikia:app